### ISSUE TITLE
Something confused about 'scale' module in denseUnet network

### ISSUE THREAD

**Issue by Alagirl** [thangylvp/HDenseUet#3]
I am confused about 'scale' module in denseUnet network.Could you tell me what 'scale' module do in the network?Thank you so much!

**Comment by thangylvp** [comment]
it is familiar with Normalization. X_new = X_old * alpha + beta.
At the initial stage, alpha = 1 beta = 0

**Comment by Alagirl** [comment]
Oh I know.Thanks a lot!

**Comment by guolihong** [comment]
Is setting the parameter "affine=True" in  nn.BatchNorm2d() the same as your ""scale?
